app/views/common/_puppetclasses_or_envs_changed.html.erb

```diff
@@ -1,3 +1,5 @@
+<%= stylesheet 'puppetclasses_or_envs_changed' %>
```

Comment:
You shouldn't need to include this, our application.css includes all other CSS recursively.